appsmithorg/appsmith
# fix: Add sheetjs package for parsing xls/xlsx file. Remove exceljs

Fixes #23405

Test Plan:

Steps to import an excel file into document viewer:
1. Import a file using file picker widget in base64 format.
2. Bind filepicker.files[0].data to document viewer data.

Test cases to cover
1. Import an .xls file into document viewer and it should work.
2. Import an .xlsx file into document viewer and it should work
3. Import an .xlsx file, verify it renders fine. Changing the file data to an .xls file. Verify it renders fine without having the need to use another document viewer widget
